src/components/DropDown/DropDown.scss

```diff
@@ -0,0 +1,49 @@
+@use "../../../src/styles/variables" as *;
+@use "../../styles/mixins" as *;
+
+.dropdown {
+  padding: 1rem;
+  width: min-content;
+
+  .dropdown__item {
+    background-color: $white;
+    min-width: 15rem;
+    border: 2px solid $black;
+    padding: 1rem;
+
+    &--primary {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      border-radius: 2rem;
+      text-align: center;
+
+      &--active {
+        border-bottom-left-radius: 0;
+        border-bottom-right-radius: 0;
+        text-align: left;
+      }
+    }
+
+    &:last-child {
+      border-bottom-left-radius: 2rem;
+      border-bottom-right-radius: 2rem;
+    }
+
+    .dropdown__item-name--primary {
+      width: 60%;
+    }
+
+    .dropdown__image-container {
+      width: 20%;
+    }
+  }
+}
+
+.arrow-active {
+  transform: rotate(180deg);
+}
```

Comment:
should this be `dropdown__arrow--active` like in the jsx?

Reply:
I dont think so. In the jsx it is: dropdown__arrow arrow-active

Reply:
either way, any classes in this file should start with `dropdown__`

Reply:
has been resolved, thank you for the clarification